### PR TITLE
refs #12147 - adding stub for rails4 compatibility

### PR DESCRIPTION
**Pull request by unorthodoxgeek**

### CODE REVIEW

File: test/lib/tasks/seeds_test.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@ class SeedsTest < ActiveSupport::TestCase
     DatabaseCleaner.clean_with :truncation
     Setting.stubs(:[]).with(:administrator).returns("root@localhost")
     Setting.stubs(:[]).with(:send_welcome_email).returns(false)
+    Setting.stubs(:[]).with('entries_per_page').returns(20)
```
Comment (dLobatog) [inline test/lib/tasks/seeds_test.rb]: I think everything comes from the fact that https://github.com/theforeman/foreman/blob/develop/lib/foreman.rb#L16 is not really detecting whether you're on a rake task or not. There's another PR I opened about this - https://github.com/theforeman/foreman/pull/2797 I should reopen with a proper fix.

**Comment by domcleal** [comment]
Do you have any more info about what this is fixing?  I can't see where entries_per_page would be used in this test.

**Comment by dLobatog** [comment]
@domcleal The issue is that since in Rails 4 `Foreman.in_rake?` seems not to return true when in `rake:test` it gets in here https://github.com/theforeman/foreman/blob/develop/lib/core_extensions.rb#L121 and if that Setting is called without expecting it, the test fails.

**Comment by domcleal** [comment]
Does `in_rake?` return true for the real db:seed then, but not when it's being tested?

**Comment by dLobatog** [comment]
@domcleal Exactly

**Comment by domcleal** [comment]
Okay, can we stub that instead please?  That way we'll at least be closer to what db:seed is actually doing I think.

The commit message also needs to explain why this is necessary, it doesn't contain any useful info right now.

**Comment by unorthodoxgeek** [comment]
@domcleal - done. sorry for the delay

**Comment by domcleal** [comment]
Merged as 3b84be4, thanks @unorthodoxgeek.

**Pull request closed without merging**